docs/labels.md

```diff
@@ -0,0 +1,106 @@
+progress, plan roadmaps, and automate issue grooming.
+
+To ensure that contributing new issues and PRs remains straight-forward, we would
+like to keep the labels required for submission to a minimum. The remaining
+labels will be added either by automation or manual grooming by other
+contributors and maintainers.
+
+This labels in this list originated within Kubernetes at
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md.
+
+## Labels that apply to issues or PRs
+
+| Label | Description | Added By |
+|-------|-------------|----------|
+| api-review                         | Categorizes an issue or PR as actively needing an API review. | Any |
+| area/api                           | Issues or PRs related to an API | Any |
+| area/blog                          | Issues or PRs related to blog entries. | Any |
+| area/conformance                   | Issues or PRs related to conformance testing | Any |
+| area/dependency                    | Issues or PRs related to dependency changes | Any |
+| area/e2e-test                      | Issues or PRs related to end-to-end testing. | Any |
+| area/endpoint/identity             | Issues or PRs related to endpoint identity | Any |
+| area/endpoint/selection            | Issues or PRs related to endpoint selection | Any |
+| area/endpoint/type                 | Issues or PRs related to endpoint type | Any |
+| area/ipam                          | Issues or PRs related to IP address management (IPAM) | Any |
+| area/licensing                     | Issues or PRs related to Antrea licensing | Any |
+| area/monitoring/logging            | Issues or PRs related to logging | Any |
+| area/monitoring/auditing           | Issues or PRs related to auditing | Any |
+| area/monitoring/traffic-analysis   | Issues or PRs related to traffic analysis | Any |
+| area/monitoring/mirroring          | Issues or PRs related to mirroring | Any |
+| area/network-policy/action         | Issues or PRs related to network policy actions | Any |
+| area/network-policy/api            | Issues or PRs related to the network policy API | Any |
+| area/network-polciy/failsafe       | Issues or PRs related to network policy failsafes | Any |
+| area/network-policy/lifecycle      | Issues or PRs related to the network policy lifecycle | Any |
+| area/network-policy/match          | Issues or PRs related to matching packets | Any |
+| area/network-policy/named-set      | Issues or PRs releatd to network policy named sets | Any |
+| area/network-policy/precedence     | Issues or PRs related to network policy precedence | Any |
+| area/network-policy/scope          | Issues or PRs related to network policy scope (namespace, global, etc.) | Any |
+| area/network-policy/staging        | Issues or PRs related to staging network policies | Any |
+| area/octant                        | Issues or PRs related to Octant | Any |
+| area/ovs/openflow                  | Issues or PRs related to Open vSwitch Open Flow | Any |
```

Comment:
if you have `area/ovs/openflow `, you may want to add `area/ovs/ovsdb` for bridge configuration issues

Reply:
great suggestion. will add it.